rest_framework/serializers.py

```diff
@@ -681,7 +681,7 @@ def to_representation(self, data):
         """
         # Dealing with nested relationships, data can be a Manager,
         # so, first get a queryset from the Manager if needed
-        iterable = data.all() if isinstance(data, models.Manager) else data
+        iterable = data.all() if isinstance(data, models.manager.BaseManager) else data
```

Comment:
I triggered the CI, can you please pull from main branch again? the bug was not verify able in the test suite, would you mind investigate further to figure out if we can add some relevant test for this?

Reply:
> I triggered the CI, can you please pull from main branch again? the bug was not verify able in the test suite, would you mind investigate further to figure out if we can add some relevant test for this?

_Yes, but it may take a while. 
I'm a little hungry. I'm going to cook the dinner first. 😋_

---

Is that what you mean? ( My English is not good )
1. Append the test case.
2. merge this branch to master in my fork ( I'm not sure about this ).
3. Then pull request again.

Reply:
I meant pull from encode:master to your working branch for this PR 954-ivory:master so that the recently updated versions in CI could reflect here. And some unit tests :)

Reply:
thanks for the rebase, now just appending/adding additional test cases would be really great